python/ray/serve/_private/utils.py

```diff
@@ -493,52 +492,6 @@ def check_obj_ref_ready_nowait(obj_ref: ObjectRef) -> bool:
     return len(finished) == 1
 
 
-serve_telemetry_tag_map = {
-    "SERVE_API_VERSION": TagKey.SERVE_API_VERSION,
-    "SERVE_NUM_DEPLOYMENTS": TagKey.SERVE_NUM_DEPLOYMENTS,
-    "GCS_STORAGE": TagKey.GCS_STORAGE,
-    "SERVE_NUM_GPU_DEPLOYMENTS": TagKey.SERVE_NUM_GPU_DEPLOYMENTS,
-    "SERVE_FASTAPI_USED": TagKey.SERVE_FASTAPI_USED,
-    "SERVE_DAG_DRIVER_USED": TagKey.SERVE_DAG_DRIVER_USED,
-    "SERVE_HTTP_ADAPTER_USED": TagKey.SERVE_HTTP_ADAPTER_USED,
-    "SERVE_GRPC_INGRESS_USED": TagKey.SERVE_GRPC_INGRESS_USED,
-    "SERVE_REST_API_VERSION": TagKey.SERVE_REST_API_VERSION,
-    "SERVE_NUM_APPS": TagKey.SERVE_NUM_APPS,
-    "SERVE_NUM_REPLICAS_LIGHTWEIGHT_UPDATED": (
-        TagKey.SERVE_NUM_REPLICAS_LIGHTWEIGHT_UPDATED
-    ),
-    "SERVE_USER_CONFIG_LIGHTWEIGHT_UPDATED": (
-        TagKey.SERVE_USER_CONFIG_LIGHTWEIGHT_UPDATED
-    ),
-    "SERVE_AUTOSCALING_CONFIG_LIGHTWEIGHT_UPDATED": (
-        TagKey.SERVE_AUTOSCALING_CONFIG_LIGHTWEIGHT_UPDATED
-    ),
-    "SERVE_RAY_SERVE_HANDLE_API_USED": TagKey.SERVE_RAY_SERVE_HANDLE_API_USED,
-    "SERVE_RAY_SERVE_SYNC_HANDLE_API_USED": TagKey.SERVE_RAY_SERVE_SYNC_HANDLE_API_USED,
-    "SERVE_DEPLOYMENT_HANDLE_API_USED": TagKey.SERVE_DEPLOYMENT_HANDLE_API_USED,
-    "SERVE_DEPLOYMENT_HANDLE_TO_OBJECT_REF_API_USED": (
-        TagKey.SERVE_DEPLOYMENT_HANDLE_TO_OBJECT_REF_API_USED
-    ),
-}
-
-
-def record_serve_tag(key: str, value: str):
```

Comment:
@shrekris-anyscale you seem to have added this. I didn't understand the docstring (not sure why `TagKey` objects would be pickled).

The tests are passing with my updated code, so let me know if I'm missing something.